docs/hub-tutorials/join-mainnet.md

```diff
@@ -215,16 +193,169 @@ enable = true
+
+Start Gaia to begin state sync. It may take take some time for the node to acquire a snapshot, but the command and output should look similar to the following:
+
+```bash
+$ gaiad start --x-crisis-skip-assert-invariants
+
+...
+
+> INF Discovered new snapshot format=1 hash="0x000..." height=8967000 module=statesync
+
+...
+
+> INF Fetching snapshot chunk chunk=4 format=1 height=8967000 module=statesync total=45
+> INF Applied snapshot chunk to ABCI app chunk=0 format=1 height=8967000 module=statesync total=45
+```
+
+Once state sync successfully completes, the node will begin to process blocks normally. If state sync fails and the node operator encounters the following error:  `State sync failed err="state sync aborted"`, either try restarting `gaiad` or running `gaiad unsafe-reset-all` (make sure to backup any configuration and history before doing this).
+
+### Quicksync
+Quicksync.io offers several  daily snapshots of the Cosmos Hub with varying levels of pruning (`archive` 1.4TB, `default` 540GB, and `pruned` 265GB). For downloads and installation instructions, visit the [Cosmos Quicksync guide](https://quicksync.io/networks/cosmos.html).
+
+
+## Snapshots
+Saving and serving snapshots helps nodes rapidly join the network.
+
+To contribute snapshots to the network, `snapshot-interval` needs to be configured in `~/.gaia/config/app.toml`. The Cosmos Hub recommends setting this value to `1000`. This value must match `pruning-keep-every` in `config.toml`.
+
+In `app.toml`
+```
+###############################################################################
+###                        State Sync Configuration                         ###
+###############################################################################
+
+# State sync snapshots allow other nodes to rapidly join the network without replaying historical
+# blocks, instead downloading and applying a snapshot of the application state at a given height.
+[state-sync]
+
+# snapshot-interval specifies the block interval at which local state sync snapshots are
+# taken (0 to disable). Must be a multiple of pruning-keep-every.
+snapshot-interval = 1000
+
+# snapshot-keep-recent specifies the number of recent snapshots to keep and serve (0 to keep all).
+snapshot-keep-recent = 10
+```
+
+It is highly recommended that node operators use the same value for snapshot-interval in order to aid snapshot discovery. Discovery is easier when more nodes are serving the same snapshots.
+
+
+## Hardware
+Running a full archive node can be resource intensive as the full  current `cosmoshub-4` state is over `1.4TB`. For those who wish to run state sync or use quicksync, the following hardware configuration is recommended:
+
+| Node Type     | RAM                   | Storage     |
+| -----------   | --------------------- | ----------- |
+| Validator     | 16GB (32 Recommended) | 500GB-2TB*  |
+| Full/Archive  | 8GB (16 Recommended)  | 2TB         |
+| Default       | 8GB (16 Recommended)  | 1TB         |
```

Comment:
this table is our recommendation, we should just put the actual number we recommend, 32, 16, 16, 8